### ISSUE TITLE
Pulmonary hypertension

### ISSUE THREAD

**Issue by tudorgroza** [obophenotype/human-phenotype-ontology#6086]
**Preferred term label:**
Pulmonary hypertension

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by LCCarmody** [comment]
Thanks for the ticket.  

Parent term: Abnormality of pulmonary circulation
@pnrobinson Question about the child terms. There are 'Pulmonary arterlial hypertension' and 'Pulmonary venous hypertension', which should be child terms.  Should 'Elevated pulmonary artery pressure' HP_0004890 be a sibling?

**Comment by pnrobinson** [comment]
In clinical usage, Pulmonary hypertension is used for the arterial variety. I will improve the definition and ask our pulmonary colleagues for input.

**Comment by pnrobinson** [comment]
The definition is also wrong because it says arterial blood, but yhe pulmonary artery carries deoxygenated blood from the right ventricle to the lungs.  @matentzn 
I also see that this term is the sole child of https://hpo.jax.org/app/browse/term/HP:0004890
Probably we should merge.
The phrase "pumonary hypertension" is often use also to describe diseases characterized by pulmonary arterial hypertension,
see https://www.mayoclinic.org/diseases-conditions/pulmonary-hypertension/symptoms-causes/syc-20350697
So probably the best label is 'Elevated pulmonary artery pressure' because it is more clearly a phenotype. I will do some research before making changes.

**Comment by matentzn** [comment]
Not sure why you tagged me on this :) If the this is about an EQ, @nicolevasilevsky should probably deal with it!

**Comment by pnrobinson** [comment]
I switched 'arterial blood' with simply blood -- the definition now seems correct
